securedrop/worker.py

```diff
@@ -1,14 +1,37 @@
-import os
-
+from flask import current_app
 from redis import Redis
 from rq import Queue
 
-queue_name = 'test' if os.environ.get(
-    'SECUREDROP_ENV') == 'test' else 'default'
 
-# `srm` can take a long time on large files, so allow it run for up to an hour
-q = Queue(name=queue_name, connection=Redis(), default_timeout=3600)
+class RqWorkerQueue(object):
+
+    '''
+    A reference to a `rq` worker queue.
+
+    Configuration:
+        `RQ_WORKER_NAME`: Name of the `rq` worker.
+    '''
+
+    __EXT_NAME = 'rq-worker-queue'
+
+    def __init__(self, app=None):
+        self.__app = app
+        if app is not None:
+            self.init_app(app)
+
+    def init_app(self, app):
+        self.__app = app
+        self.__app.config.setdefault('RQ_WORKER_NAME', 'default')
+
+        if not hasattr(self.__app, 'extensions'):
```

Comment:
If feel like this pattern feels weird

```python
try:
    self.__app.extensions
except AttributeError:
    self.__app.extensions = {}
```

Like because we don't actually use the value it seems wrong.

Reply:
```Python
try:
    _ = self.__app.extensions  # To check the existence of the variable
except AttributeError:
    self.__app.extensions = {}
```

What about this way?

Reply:
(relevant: https://hynek.me/articles/hasattr/)

Reply:
Lol wow I thought that @kushaldas was mostly making a note about it being more pythonic. I never knew that checking for an existence of an attribute could raise errors in Python 3. :neutral_face: 

I can make this change later.

Reply:
```Python
>>> d = {}
>>> _ = d.not_there
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict' object has no attribute 'not_there'
```